great_expectations/rule_based_profiler/parameter_builder/simple_date_format_string_parameter_builder.py

```diff
@@ -6,18 +6,18 @@
 from great_expectations.rule_based_profiler.helpers.util import (
     get_parameter_value_and_validate_return_type,
 )
-from great_expectations.rule_based_profiler.parameter_builder import (
-    AttributedResolvedMetrics,
-    MetricComputationResult,
-    MetricValues,
-    ParameterBuilder,
-)
+from great_expectations.rule_based_profiler.parameter_builder import ParameterBuilder
 from great_expectations.rule_based_profiler.types import (
     FULLY_QUALIFIED_PARAMETER_NAME_METADATA_KEY,
     FULLY_QUALIFIED_PARAMETER_NAME_VALUE_KEY,
     Domain,
+    MetricComputationResult,
+    MetricValues,
     ParameterContainer,
 )
+from great_expectations.rule_based_profiler.types.attributed_resolved_metrics import (
```

Comment:
@Shinnnyshinshin In my opinion, these should be accessible from `types` -- hence, defined in that directory's `__init__.py` file.  Could you please give it a shot and see if it will work? Alternatively, if you see a counter-pattern that you agree with more, please let me know.  Otherwise, seems all good.  Thanks!

Reply:
I tried this but it gave me a circular import error from this line in the `attributed_resolved_metrics.py` file. 
I'll have another stab at it in the morning. I like the idea and would like to see if it is possible. 

```
from great_expectations.rule_based_profiler.types import MetricValue, MetricValues
```